spec/services/claims/claim/remove_internal_drafts_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+describe Claims::Claim::RemoveInternalDrafts do
+  subject(:service) { described_class.call }
+
+  let!(:draft_claim) { create(:claim, status: :draft) }
+  let!(:submitted_claim) { create(:claim, status: :submitted) }
+
+  it_behaves_like "a service object"
+
+  describe "#call" do
+    it "removes internal draft claims updated 2 hours ago" do
+      create(:claim, status: :internal_draft, updated_at: 3.hours.ago)
+
+      expect { service }.to change(Claims::Claim, :count).from(3).to(2)
+      expect(Claims::Claim.all).to eq([draft_claim, submitted_claim])
```

Comment:
Can we add an `internal_draft` claim that was recently updated to this list too please?